ert_storage/database_schema.py

```diff
@@ -1,13 +1,16 @@
 from enum import Enum
-from typing import Any, Mapping, Optional
+from typing import Any, Optional, Mapping
 
 import sqlalchemy as sa
+import uuid
```

Comment:
Style: maybe we should do `from uuid import UUID`? There's a lot of `uuid.UUID` in this PR which looks wonky to me.